jme3-core/src/main/java/com/jme3/renderer/opengl/GLDebug.java

```diff
@@ -46,4 +97,33 @@ protected void checkError() {
             throw new RendererException("An OpenGL error occurred - " + decodeError(err));
         }
     }
+
+    @Override
+    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
+        Object result = method.invoke(obj, args);
+
+        if (method.getName().equals("glGetError")) {
```

Comment:
@pspeed42 Actually I just stumbled over this:
Do you think it would be more wise to lookup the Method object and use equals there instead of comparing the string all the time?

Reply:
Sure it would be slightly faster, I guess, if the method is looked up at proxy creation time.  I wasn't overly concerned about it since this is a debugging class.  

I'm ok if that's a follow-on improvement, too.